rootfs/usr/share/inputplumber/devices/50-gpd_win4.yaml

```
@@ -36,8 +37,24 @@ source_devices:
     iio:
       name: i2c-BMI0160:00
       mount_matrix:
-        # Tested on 6800U
-        # TODO: Verify on other models
+        x: [-1, 0, 0]
+        y: [0, -1, 0]
+        z: [0, 0, 1]
+  # GPD Win4 2025 (HX 370)
+  - group: gamepad
+    evdev:
+      name: "Microsoft X-Box 360 pad"
+      phys_path: usb-0000:66:00.0-2.1/input0
+      handler: event*
+  - group: keyboard
+    evdev:
+      name: "  Mouse for Windows"
+      handler: event*
+      phys_path: usb-0000:66:00.0-2.2/input1
+  - group: imu
+    iio:
+      name: i2c-BMI0260:00
```

To match both the out-of-tree and in-tree devices, you can make this change:

```suggestion
      name: {i2c-BMI0260:00,bmi260}
```